### PR TITLE
Update workflow --help texts

### PR DESCRIPTION
**Pull request by feedmeapples**
## What was changed

Updated the --help texts of `tctl workflow` commands

## Why?

attempt to make it nicer to read

## Checklist

1. Closes 

2. How was this tested:

3. Any docs updates needed?

### CODE REVIEW

**Comment by alexshtin** [comment]
LGTM, but I would request another round from @flossypurse.

**Review by alexshtin — APPROVED** [review]

File: cli/workflow.go [hunk 1]
```diff
@@ -52,7 +52,7 @@ func newWorkflowCommands() []*cli.Command {
 		{
 			Name:    "describe",
 			Aliases: []string{"d"},
-			Usage:   "Show information about a Workflow Execution",
+			Usage:   "Show a Workflow Execution info",
```
Comment (djmagee) [inline cli/workflow.go:55]: I recommend keeping the original wording. (I'm not really sure what "an info" is.)

File: cli/workflow.go [hunk 2]
```diff
@@ -70,23 +70,23 @@ func newWorkflowCommands() []*cli.Command {
 		{
 			Name:    "list",
 			Aliases: []string{"l"},
-			Usage:   "List Workflow Executions based on query",
+			Usage:   "List Workflow Executions based on a query",
```
Comment (djmagee) [inline cli/workflow.go:73]:
```suggestion
			Usage:   "List Workflow Executions based on a Query",
```

File: cli/workflow.go [hunk 3]
```diff
@@ -70,23 +70,23 @@ func newWorkflowCommands() []*cli.Command {
...
 			Flags:   append(flagsForWorkflowFiltering, flags.FlagsForPaginationAndRendering...),
 			Action: func(c *cli.Context) error {
 				return ListWorkflow(c)
 			},
 		},
 		{
 			Name:  "show",
-			Usage: "Show Workflow Execution history",
+			Usage: "Show a Workflow Execution history",
```
Comment (djmagee) [inline cli/workflow.go:81]:
```suggestion
			Usage: "Show Event History for a Workflow Execution",
```

Comment (feedmeapples) [reply cli/workflow.go:81]: ty

**Pull request merged**